programs/github

```diff
@@ -79,6 +79,43 @@ if flags.branch and flags.tag then
 	return printError('--branch and --tag cannot both be specified')
 end
 
+
+local function hasEnoughSpace(repoSize, freeSpace)
+	-- warn if repository will fill more than 80% of the
+	-- current freeSpace.
+	local sizeError = 0.2
```

Comment:
This comment isn't even true! It's 83%.

I means something like "The value reported by github doesn't match the one reported by CC". Summarize the conclusions in your imgur link in the comment.